src/broker/cas_cgw.c

```diff
@@ -1123,6 +1128,9 @@ cgw_set_bindparam (T_CGW_HANDLE * handle, int bind_num, void *net_type, void *ne
 	size_t precision, scale;
 	char num_str[64];
 	char tmp[64];
+	SQLHDESC hdesc = NULL;
+
+	SQL_CHK_ERR (handle->hdbc, SQL_HANDLE_DBC, err_code = SQLAllocHandle (SQL_HANDLE_DESC, handle->hdbc, &hdesc));
```

Comment:
no necessary to check hdesc is null?

Reply:
If assignment from SQLAllocHandle to desc fails, an error occurs. At this time, it moves to goto to handle the error.